src/components/PageWithSideMenu.tsx

```diff
@@ -49,7 +49,7 @@ export const SideNavHeader = ({ icon, text, id }: SideNavHeaderProps) => {
   const IconComponent = icon;
   return (
     <StyledSideMenuHeader>
-      {IconComponent && <IconComponent fontSize="large" />}
+      {IconComponent && <IconComponent sx={{ height: '1.25rem' }} />}
```

Comment:
skal dette egentlig være fontSize?
```suggestion
      {IconComponent && <IconComponent sx={{ fontSize: '1.25rem' }} />}
```